buildbot/tox_setup.sh

```diff
@@ -1,74 +1,102 @@
 #!/usr/bin/env bash
-# pyenv root
-export PYENV_ROOT="$HOME/.pyenv"
-TEST_ROOT=$PWD/..
 
-# Install pyenv if it's missing
-if [[ ! -d $PYENV_ROOT ]]; then
-    git clone git://github.com/yyuu/pyenv.git ${PYENV_ROOT}
-    cd ${PYENV_ROOT}
-    # Get the latest tagged version
-    git checkout `git tag | tail -1`
+if [[ ! -d $PYENV_ROOT ]]
+then
+    export PYENV_ROOT="$HOME/.pyenv"
 fi
 
-# Upgrade it, if it's too old
-if [[ -z $(pyenv install --list | grep 3.4.3) ]]; then
-    cd ${PYENV_ROOT}
-    git pull origin master
-    git pull -u origin master
-    # Get the latest tagged version
-    git checkout `git tag | tail -1`
+declare -r PROJDIR="$PWD/.."
+if [[ ! -s $PROJDIR/riak/__init__.py ]]
+then
+    echo "[ERROR] script must be run from the root of a clone of github.com/basho/riak-python-client" 1>&2
```

Comment:
You could just cd to `PROJDIR` for the remainder of the script, right? Not a biggie but I have run it from other places to set up multiple Pythons since it's rather handy to have this already pre-scripted.